playbooks/ORCHESTRATION.md

```diff
@@ -127,9 +127,70 @@ Manually check for:
 
 <!-- TODO: Add playbook for disaster/recovery steps. -->
 
-### Hardware Key Signing
+## Step 3: Add delegation (Optional)
+
+This step will add a delegated role to the top-level targets that is
+controlled by an external GitHub repository. Coordinate with the
+delegation keyholder to run the `add-delegation` (see
+https://github.com/sigstore/root-signing/blob/main/cmd/tuf/app/add-delegation.go).
+When creating the delegation with the command, a `target-meta`
+file has to be provided that lists the targets, similar to adding
+the top level targets.
+After the delegation metadata is added and signed, the delegation
+keyholder should open a PR against the ceremony branch.
+The name of the PR MUST be `feat/add-delegation for
+<delegation-name>`.
+
+As part of running the `add-delegaton` command, a POP (proof of
+possession) has to be generated too. The computed POP should be stored
+in `${REPO}/staged/${FORK_POINT}.sig`, where the fork point is the
```

Comment:
when you say "should be stored in" does that mean the person following these instructions has to do something? if this is done automatically by one of the commands below, we might say "will be stored in ..."

Reply:
Currently there is no automation done for this, but it's shown in the stanza below:
```
$ FORK_POINT=$(git merge-base --fork-point origin/main "${BRANCH}") \
      ./tuf key-pop-sign \
      -key ${KEY_REF} \
      -challenge ${DELEGATION_NAME} \
      -nonce ${FORK_POINT} > ${REPO}/staged/${FORK_POINT}.sig
```

Reply:
> Currently there is no automation done for this, but it's shown in the stanza below:

This is probably sufficient for now, thanks.

Reply:
The example in the documentation should show all steps required to create a delegation, and then the delegate owner could open a PR with the change-set to have it merged. So there should be no uncertainties, if so let me know and I will update the doc 😄